### ISSUE TITLE
replication task and schedule for snapshots both disappeared

### ISSUE THREAD

**Issue by kmoore134** [trueos/trueos-core#68]
_From @grahamperrin on August 26, 2016 19:47_

I don't know when they disappeared, but a FreeNAS view of things shows that the last replication was a few days ago: 

```
[ergovista621@freenas] ~% zfs list -t snapshot | grep gjp4
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-01-30-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-01-35-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-01-40-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-01-45-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-01-50-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-01-55-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-00-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-05-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-10-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-15-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-20-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-25-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-30-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-35-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-40-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-45-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-50-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-02-55-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-00-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-05-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-10-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-15-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-20-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-25-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-30-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-35-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-40-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-45-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-50-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-03-55-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-00-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-05-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-10-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-15-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-20-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-25-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-30-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-35-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-40-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-45-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-50-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-04-55-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-00-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-05-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-10-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-15-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-20-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-25-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-30-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-35-00     8K      -  6.94G  -
tallhalf/pcbsd/gjp4/personacrypt/momh167-gjp4-macbookpro112-trueos.university.brighton.ac.uk@auto-2016-08-22-05-40-00      0      -  6.94G  -
[ergovista621@freenas] 
```

_Copied from original issue: trueos/lpreserver#3_

**Comment by kmoore134** [comment]
I'm not sure whats going on here. Is this still an issue? Do you have logs / details from the sending side?

**Comment by kmoore134** [comment]
_From @grahamperrin on September 29, 2016 19:58_

Hard to tell what's going on. I tried to change the replication frequency from sync to manual, that change was not effective and now in the Dataset field I see: 

tallhalf/pcbsd/gjp4/personacrypt Time

I ought to take a look after the next round of updates.

**Comment by grahamperrin** [comment]
> Hard to tell what's going on. 

Yep. 

If the problem recurs, then I might give more thought to BEs e.g. … whether a perceived loss was the result of preferring an older (less recently created) BE after setting a schedule in a newer but somehow inferior BE. 

----

Side note: a few days ago I found a duplicate zfsm line in the anacron section of my `/etc/crontab`. Harmless, I guess, but for tidiness I removed the duplicate.
